tools/test_parser_generator.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Validate that the current version of the generated parser matches the output
+# generated by the version of goyacc installed on the local system.
+#
+# This is used in travis to verify that the currently committed version was
+# generated with the proper version of goyacc.
+
+CUR="sql.go"
+TMP="/tmp/sql.$$.go"
+
+
+if ! cd go/vt/sqlparser/
```

Comment:
nit: Let's agree on having "; then" on the same line as the if.

Same comment below.